### ISSUE TITLE
Unable to use programmatic API since upgrading from 0.6.0

### ISSUE THREAD

**Issue by larrifax** [RobinBlomberg/kysely-codegen#42]
I just upgraded from 0.6.0, and am currently unable to use the programmatic API, like so:

```
  new Generator().generate({
    camelCase: false,
    connectionString: produceConnectionString(),
    dialect: new DialectManager().getDialect("postgres"),
    outFile: OUTPUT_PATH,
  });
```

This is the error message I get when running it:
```
• Loaded environment variables from .env file.
2022-10-24T09:17:04.410Z [ERROR]: unhandledRejection: Environment variable 'DATABASE_URL' could not be found.
/workspace/node_modules/kysely-codegen/dist/connection-string-parser.js:47
                throw new ReferenceError(`Environment variable '${key}' could not be found.`);
                      ^

ReferenceError: Environment variable 'DATABASE_URL' could not be found.
    at ConnectionStringParser.parse (/workspace/node_modules/kysely-codegen/dist/connection-string-parser.js:47:23)
    at Cli._Cli_generate (/workspace/node_modules/kysely-codegen/dist/cli.js:99:78)
    at Cli.run (/workspace/node_modules/kysely-codegen/dist/cli.js:90:80)
    at <anonymous> (/workspace/node_modules/kysely-codegen/dist/bin/index.js:5:22)
    at Object.<anonymous> (/workspace/node_modules/kysely-codegen/dist/bin/index.js:5:47)
    at Module._compile (node:internal/modules/cjs/loader:1155:14)
    at Object.v (/workspace/node_modules/@esbuild-kit/cjs-loader/dist/index.js:1:901)
    at Module.load (node:internal/modules/cjs/loader:1033:32)
    at Function.Module._load (node:internal/modules/cjs/loader:868:12)
    at Module.require (node:internal/modules/cjs/loader:1057:19)
2022-10-24T09:17:04.410Z [ERROR]: unhandledRejection: Environment variable 'DATABASE_URL' could not be found.
/workspace/node_modules/kysely-codegen/dist/connection-string-parser.js:47
                throw new ReferenceError(`Environment variable '${key}' could not be found.`);
                      ^

ReferenceError: Environment variable 'DATABASE_URL' could not be found.
    at ConnectionStringParser.parse (/workspace/node_modules/kysely-codegen/dist/connection-string-parser.js:47:23)
    at Cli._Cli_generate (/workspace/node_modules/kysely-codegen/dist/cli.js:99:78)
    at Cli.run (/workspace/node_modules/kysely-codegen/dist/cli.js:90:80)
    at <anonymous> (/workspace/node_modules/kysely-codegen/dist/bin/index.js:5:22)
    at Object.<anonymous> (/workspace/node_modules/kysely-codegen/dist/bin/index.js:5:47)
    at Module._compile (node:internal/modules/cjs/loader:1155:14)
    at Object.v (/workspace/node_modules/@esbuild-kit/cjs-loader/dist/index.js:1:901)
    at Module.load (node:internal/modules/cjs/loader:1033:32)
    at Function.Module._load (node:internal/modules/cjs/loader:868:12)
    at Module.require (node:internal/modules/cjs/loader:1057:19)
```

It seems like the culprit could be the following, as removing `__exportStar(require("./bin"), exports);` from `dist/index.js` fixes the problem
https://github.com/RobinBlomberg/kysely-codegen/blob/f4cc73f2496ee99b92aa6ca0709d22d93a41a77b/src/index.ts#L2

**Comment by RobinBlomberg** [comment]
Should be fixed in `kysely-codegen@0.8.0` now!

**Comment by virtuallyunknown** [comment]
Hey @RobinBlomberg, I don't want to open a separate issue, but is there any chance you could mention somewhere in the docs that you can use the `kysely-codegen` via API instead of the command line?

I was completely unaware of this fact and was trying to use node `child_process` to spawn/exec `kysely-codegen` because I'm working within a monorepo and can't directly access `.env`, but that lead to all kinds of other problems.

I happened to find this thread completely by chance while looking at the list of issues.

Cheers!
